assets/i18n/en.json

```diff
@@ -554,9 +554,11 @@
     "I18N_TOPNAV_ABOUT_OPPIA": "About Oppia",
     "I18N_TOPNAV_ADMIN_PAGE": "Admin Page",
     "I18N_TOPNAV_BLOG": "Blog",
+    "I18N_TOPNAV_CLASSROOM": "Classroom",
+    "I18N_TOPNAV_CLASSROOM_BASICMATHS": "Basic Mathematics",
     "I18N_TOPNAV_CONTACT_US": "Contact Us",
     "I18N_TOPNAV_CREATOR_DASHBOARD": "Creator Dashboard",
-    "I18N_TOPNAV_DONATE": "Donate",
+    "I18N_TOPNAV_DONATE": "Donate",    
```

Comment:
Please remove this trailing whitespace. The lint test should be failing, I hope.

Reply:
I will check this but while pushing the changes It said All checks passed.